app/src/main/java/com/example/levi9application/viewModel/CocktailViewModel.kt

```diff
@@ -31,18 +38,26 @@ class CocktailViewModel
 
     init {
         getCocktails()
+        val cocktailDAO = CocktailDatabase.getDatabase(application).getDao()
+        _repository = CocktailDataRepo(cocktailDAO)
```

Comment:
Since we are using Hilt, repos should be injected and not instantiated like this